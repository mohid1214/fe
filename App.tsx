import React from 'react';
import {FlatList, TouchableHighlight, View} from 'react-native';
import {SafeAreaProvider} from 'react-native-safe-area-context';
import {StyleSheet} from 'react-native';
import {Registration} from './Screens/Registration/Registration';

const App = () => {
  return (
    <SafeAreaProvider>
      <View style={{flex: 1}}>
        <Registration />
      </View>
    </SafeAreaProvider>
  );
};

export default App;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
  overlayContainer: {
    width: 300,
  },
});
