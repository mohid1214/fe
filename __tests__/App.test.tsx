
import React from 'react';
import App from '../App';



// Note: import explicitly to use the types shipped with jest.
import {it,expect,test,describe} from '@jest/globals';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';






