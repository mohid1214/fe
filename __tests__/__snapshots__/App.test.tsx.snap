// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "blue",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
      {
        "backgroundColor": "#ef4444",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#22c55e",
        },
        {
          "flexBasis": "0%",
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    Tai here
  </Text>
</View>
`;
